go-memory-allocation/README.md

```
@@ -0,0 +1,1198 @@
+  - [Arena and Page](#arena-and-page)
+  - [Span and Size Class](#span-and-size-class)
+  - [Span Class](#span-class)
+  - [Span Set](#span-set)
+  - [Heap Bits and Malloc Header](#heap-bits-and-malloc-header)
+- [Heap Management](#heap-management)
+  - [Span Allocation: `mheap.alloc`](#span-allocation-mheapalloc)
+  - [Central Span Manager: `mcentral`](#central-span-manager-mcentral)
+  - [Processor's Memory Allocator: `mcache`](#processors-memory-allocator-mcache)
+- [Heap Allocation](#heap-allocation)
+  - [Tiny Objects: `mallocgcTiny`](#tiny-objects-mallocgctiny)
+  - [Small Objects: `mallocgcSmall*`](#small-objects-mallocgcsmall-)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)